### PR TITLE
feat(geojson): support for tracksViewChanges in Geojson component

### PR DESCRIPTION
**Pull request by jcuffney**
### Does any other open PR do the same thing?

No.

### What issue is this PR fixing?

- [x] allows for consumers to set `tracksViewChanges` for all makers via the `Geojson` component
- [x] allows for consumers to override the default `tracksViewChanges` on a per marker basis with a value set in their "point" at `properties.tracksViewChanges`

### How did you test this PR?

- This change affects both Android and iOS

### CODE REVIEW

**Comment by Simon-TechForm** [comment]
Looks good :)

Can I have you add the prop to the [api docs](https://github.com/react-native-maps/react-native-maps/blob/master/docs/geojson.md) as well?

File: src/Geojson.tsx [hunk 1]
```diff
@@ -198,11 +211,13 @@ const Geojson = (props: GeojsonProps) => {
     <React.Fragment>
       {pointOverlays.map((overlay, index) => {
         const markerColor = getColor(color, overlay, 'marker-color');
-
+        const pointOverlayTracksViewChanges =
+          overlay?.feature?.properties?.tracksViewChanges || tracksViewChanges;
```
Comment (Simon-TechForm) [inline src/Geojson.tsx:215]: Optional chaining should only be needed on the `properties` key.

**Comment by jcuffney** [comment]
> Looks good :)
> 
> Can I have you add the prop to the [api docs](https://github.com/react-native-maps/react-native-maps/blob/master/docs/geojson.md) as well?

yep!

File: docs/geojson.md [hunk 2]
```diff
@@ -18,6 +18,7 @@
 | `onPress` | `Function`     |  | returns the selected overlay value with the onPress functionality |
 | `markerComponent` | `React Node`     |  | Component to render in place of the default marker when the overlay type is a `point` 
 | `title` | `string`     |  | The title of the marker. This is only used if the <Marker /> component has no children that  are a `<Callout />` 
+| `tracksViewChanges` | `Boolean` | | Sets whether this marker should track view changes. It's recommended to turn it off whenever it's possible to improve custom marker performance. This is the default value for all point markers in your geojson data. It can be overriden on a per point basis by adding a `trackViewChanges` property to the `properties` object on the point.
```
Comment (Simon-TechForm) [inline docs/geojson.md:21]: Can we add the default value as well? `tracksViewChanges` defaults to `true`.

Comment (jcuffney) [reply docs/geojson.md:21]: yep! - just added that in.

Comment (Simon-TechForm) [reply docs/geojson.md:21]: Hmm, I don't see any changes? 🤔 Did you push your commit? :)

Comment (Simon-TechForm) [reply docs/geojson.md:21]: I'll push a doc update after merging this, so we can move forward with the PR.

**Review by Simon-TechForm — APPROVED** [review]

**Pull request merged**